HLeithner/joomla-cms
# Add prepared statements for mod_articles_archive

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the module in all ways you can think of.

* create a frontend module
* create articles with the condition archvied
* The module should show you the articles as without the PR

### Expected result
Nothing changed.
